test/test_allocation_domain.cpp

```diff
@@ -1053,7 +1053,7 @@ TEST_F(AllocationDomainTest, NHWC2d_To_NHWC2d_cacheFork_CUDA) {
   testValidate(&fusion, cg_outputs, {t0}, {t0, t0}, __LINE__, __FILE__);
 }
 
-TEST_F(NVFuserTest, AllocationDomainVectorizationIssue902) {
+TEST_F(AllocationDomainTest, AllocationDomainVectorizationIssue902) {
```

Comment:
nit: Should this be?
```suggestion
TEST_F(AllocationDomainTest, VectorizationIssue902) {
```